<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="kanakadurga" failures="1" tests="3" name="Microsoft Edge Test" time="14.781" errors="0" timestamp="2025-10-15T16:28:06 CDT">
  <testcase classname="com.lambdatest.LambdaTestNGTest" name="verifyPageTitle" time="0.353">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [LambdaTest] but found [Selenium Grid Online | Run Selenium Test On Cloud]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [LambdaTest] but found [Selenium Grid Online | Run Selenium Test On Cloud]
at com.lambdatest.LambdaTestNGTest.verifyPageTitle(LambdaTestNGTest.java:92)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- verifyPageTitle -->
  <testcase classname="com.lambdatest.LambdaTestNGTest" name="validateCheckbox" time="1.612"/>
  <testcase classname="com.lambdatest.LambdaTestNGTest" name="validateAlert" time="2.089"/>
</testsuite> <!-- Microsoft Edge Test -->
